main.js

```diff
@@ -53,9 +53,9 @@ form.addEventListener('submit', (event) => {
   }
 });
 
-const showDate = document.getElementById('showDate');
-const updateDate = showTime();
+function showTime() {
+  const dateSelector = document.getElementById('showDate');
+  dateSelector.textContent = DateTime.now().toFormat('LLLL d yyyy, hh:mm:ss a');
+}
```

Comment:
- [ ] The keyword `function` is depreciated in the new javascript ES6 syntax and is replaced by the arrow functions. So I would suggest you to use  `arrow functions` instead of function keyword as well. 

**Example snippet:**

```js
const showTime = ()  => {
...
}

```

Note: This requirement is for your entire `main.js` as I have noticed you are still using `function` keyword for your other functions like` showBooksList(event)`, `showContact(event)` and `showAddBook(event)`